code/go/0chain.net/core/config/globals.go

```diff
@@ -268,8 +268,8 @@ func initGlobalSettings() {
 	}{
 		GlobalSettingName[State]:        {Boolean, false},
 		GlobalSettingName[Dkg]:          {Boolean, false},
-		GlobalSettingName[ViewChange]:   {Boolean, false},
-		GlobalSettingName[BlockRewards]: {Boolean, false},
+		GlobalSettingName[ViewChange]:   {Boolean, true},
```

Comment:
view change should be `false` by default, we will change it to true when it's fully supported.